examples/openpilot/compile2.py

```diff
@@ -149,7 +149,8 @@ def to_ref(b:Buffer): return struct.pack("Q", id(b)).decode("latin_1")
         })
         if needs_load:
           t = Tensor.empty(b.dtype.shape, dtype=b.dtype)
-          t.lazydata.buffer = b
+          # TODO: what does this do? uop is immutable
+          #t.lazydata.buffer = b
```

Comment:
It's used in the export. You want to replace this with setting the buffer in the dicts, but really we should just delete compile2